Update Expert Docs for 2.32 (#7660)
FlowFuse/flowfuse: docs/user/expert/chat.md

The more specific your prompt, the closer the result will be to what you need. Include node names, topic paths, endpoints, or field names where you have them. Expert will use them directly rather than substituting placeholders.

> > **Availability:** This feature is available on **FlowFuse Cloud** (Starter, Team, and Enterprise tiers) and, from v2.32, to self-hosted users. On FlowFuse Cloud it must be enabled for your team on request. [Contact us](https://flowfuse.com/contact-us/?subject=FlowFuse%20Expert%20Application%20Building) to get access.

Reviewer: ```suggestion
> **Availability:** This feature is available on **FlowFuse Cloud** for all tiers, and from v2.32, to Self-Hosted Enterprise tiers. For Self-Hosted Enterprise tiers it requires a configuration for your instance. [Contact us](https://flowfuse.com/contact-us/?subject=FlowFuse%20Expert%20Application%20Building) to get access.
```

Reviewer: @sumitshinde-84

- **Always deny:** skip it and don't ask again for it for the rest of this chat.

![FlowFuse Expert pausing for approval mid-build, showing the Adding Nodes action awaiting your decision](../images/assistant/expert-tool-approval-card.png){data-zoomable}

Expert waits as long as you need, with no timeout on the decision. Stopping the chat while a card is open cancels the pending action (treated as a denial). "Always allow" and "Always deny" choices apply only to the current conversation and reset when you use **Start Over** or refresh. If you want to keep one, click **Make permanent** to save that choice for future chats.

Once you approve an action, the card collapses to show what was decided, and the build continues.

![The chat after actions have been approved, each card showing its action type and that it is allowed for the current chat](../images/assistant/expert-tool-approval-chat.png){data-zoomable}

**Configuring permissions in settings.** You can set your team's permissions ahead of time from the Expert settings dialog. To open it:

1. Open the FlowFuse Expert panel and select **Support** mode.
2. Click the settings (gear) icon at the top-right of the composer.

> Permissions are saved **per team** and persist across your chats in that team, so each team can have its own policy. Switching to another team starts from the defaults again.

Reviewer 2: ```suggestion
Permissions are saved per user, for the team you currently have active. If you switch to another team, your settings there are kept separately, so you can maintain a different policy per team without losing your choices when you switch back.
```
